casbin/casbin.go

```diff
@@ -109,7 +116,7 @@ func MiddlewareWithConfig(config Config) echo.MiddlewareFunc {
 // GetUserName gets the user name from the request.
 // Currently, only HTTP basic authentication is supported
```

Comment:
Please don't forget to update this comment (thanks to you this method supports **"to infinity and beyond!"** :wink:)